trunk/NDHMS/Routing/module_channel_routing.F

```diff
@@ -146,10 +146,10 @@ end function DIFFUSION
 ! ----------------------------------------------------------------
 
 subroutine SUBMUSKINGCUNGE(    &
-     qdc, vel, idx, qup,  quc, &
+     qdc, vel, qloss, idx, qup,  quc, &
```

Comment:
For this subroutine, qloss is after vel, and ChannK is after depth. This is the same as how David Yates coded it in NWM v1.2.